native/packages/react-native-bpk-component-badge/src/BpkBadge.js

```diff
@@ -97,6 +100,15 @@ const styles = StyleSheet.create({
     borderBottomLeftRadius: borderRadiusSm,
     borderBottomRightRadius: 0,
   },
+  accessoryView: {
+    marginEnd: spacingSm,
+  },
+  accessoryViewDestructive: {
+    color: colorWhite,
+  },
+  accessoryViewOutline: {
+    color: colorWhite,
+  },
```

Comment:
Can't we reuse `textBase`, `textDestructive` and `textOutline` styles for the accessory view?

Reply:
Yo - I tried this but Flow starts complaining at me 😢